src/request-api.js

```diff
@@ -1,108 +1,123 @@
-var http = require('http')
-var qs = require('querystring')
-var getFilesStream = require('./get-files-stream')
+'use strict'
```

Comment:
I've read that this is required for 'let' to work properly, is it the reason why you added?